seq2seq/third_party/trainers/trainer.py

```diff
@@ -28,17 +28,16 @@
 import warnings
```

Comment:
This comment refers to line 3. Similarly, this is a modified version of the original HuggingFace file so we need to indicate it has been changed as described in the doc I sent you.

Reply:
thanks, done